src/ucp/core/ucp_ep.c

```diff
@@ -2378,17 +2379,11 @@ void ucp_ep_proto_reqs_purge(ucp_ep_h ucp_ep, ucs_status_t status)
         ucp_request_send_untrack(req);
         ucp_ep_req_purge(ucp_ep, req, status, 0);
     }
-}
-
-void ucp_ep_reqs_purge(ucp_ep_h ucp_ep, ucs_status_t status)
-{
-    uct_ep_h wireup_ep;
-    ucp_request_t *req;
-
-    ucp_ep_proto_reqs_purge(ucp_ep, status);
 
-    if (ucp_ep->flags & (UCP_EP_FLAG_ON_MATCH_CTX | UCP_EP_FLAG_CLOSE_REQ_VALID)) {
+    if (/* no need to purge flush requests */
+        !purge_flush_reqs && 
```

Comment:
should it be `||`?

Reply:
yes, it was already fixed in the last commit

Reply:
did you forget to push gain?

Reply:
> did you forget to push gain?

yes :) pushed now